docs/basics/101-136-cheatsheet.rst

```diff
@@ -0,0 +1,187 @@
+.. _cheat:
+
+The DataLad Cheat-Sheet
+-----------------------
+
+DataLad is a data management and publication multitool based
+on Git and Git-annex with a command line interface and a Python
+API. With DataLad, you can version control arbitrarily large data,
+share or consume data, and record changes made to data.
+
+**Basic command structure**:
+
+``datalad [-c <config-option>] COMMAND [--flag <optional flag specification>] [PATH]``
+
+General flags for the command line interface
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following general flags can be supplied to most commands
+
+- ``-d/--dataset``: A path that points to the root of the dataset an operation is performed on.
+- ``-D/--description``: A description of the location (e.g. "my backup server").
+- ``-f/--force``: Force execution of a command (Dangerzone!).
+- ``-h/--help``: Call any command with this flag to get help.
+- ``-m/--message``: A description about a change made to the dataset.
+- ``-r/--recursive``: Perform an operation recursively in potential subdatasets.
+- ``-R/--recursion limit <n>``: Limit recursive operations to *n* levels of subdatasets.
+
+Commands
+^^^^^^^^
+
+If general flags can be applied to the commands below, this is indicated
+by a lower-case letter in front of the command name.
+
+Dataset operations
+""""""""""""""""""
+
+   +--------------+---------------+----------------------------------------------------------+
+   | Command      |General flags  |    Description                                           |
+   +==============+===============+==========================================================+
+   |              |  [h/f/d/D]    | create a new dataset from scratch.                       |
```

Comment:
All commands have help, so IMHO not with listing for each. Moreover --help is more of an action than an option - it causes actual command not do anything and just spit out help

Reply:
True, thx